lang/typescript/typescriptreact.talon

```diff
@@ -1,3 +1,4 @@
 tag: user.typescriptreact
 -
 tag(): user.typescript
+tag(): user.javascriptreact
```

Comment:
@AndreasArvidsson just wanted to check that there wasn't some reason your original PR didn't include this tag.  Is it a problem that we get a diamond inheritance structure where we activate `javascript` twice?

Reply:
Nah no problem like that with tags